conda-forge.yml

```diff
@@ -1,2 +1,2 @@
 max_py_ver: '37'
```

Comment:
Python 3.8?

Reply:
those are conda-forge defines that shouldn't be changed in PR. only base recipe